etc/dwh.yaml

```diff
@@ -0,0 +1,22 @@
+address: "127.0.0.1:15030"
+
+storage:
+  driver: "sqlite3"
+  endpoint: "target/dwh.db"
+
+# blockchain-specific settings.
+ethereum:
+  # path to keystore
+  key_store: "./keys"
+  # passphrase for keystore
+  pass_phrase: "any"
+
+
+blockchain:
+  eth_endpoint: "0x0"
+  gas_price: 10
```

Comment:
I think this setting is bit meaningless. Gas price in our private blockchain is always 0 (as @Sokel says).
`eth_endpoint` is a contract address? I think endpoint must be presented by something like URL. We already have some options into `blockchain/options.go`.